about.html

```diff
@@ -108,8 +108,10 @@ <h2 class="heading">
       </section>
       <section class="past-conf">
         <h2 class="heading">See the past conf <span></span></h2>
-        <p id="conf-1">2015 Conference</p>
-        <p id="conf-2">2018 Conference</p>
+        <div class="past-conf-images">
+          <p id="conf-1">2015 Conference</p>
+          <p id="conf-2">2018 Conference</p>
+        </div>
       </section>
       <footer class="footer">
         <img src="./assets/images/icons/flash.png" alt="flash" />
```

Comment:
- [ ] Kindly at the about page on the mobile version, the `partners' section exists, it does not exist on the desktop version but on the mobile version exists. These kinds of differences maybe appear as small ones but they will make you more comfortable with media query and how to deal with different designs. 

Your design
![image](https://user-images.githubusercontent.com/35535130/163073016-07f32daa-5fb3-42a4-8541-2b97315411a9.png)


Original design
![image](https://user-images.githubusercontent.com/35535130/163072977-1f84f4b2-fb52-47fe-b635-06f2c96d1bb1.png)

Reply:
I get it @keroloussamy, I do not know why the **partners section** is not showing on your side because on my computer when I try to access the live link, that section shows up.

Reply:
It is showing. nice work